plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -565,6 +567,48 @@ private function set_search_args(): void {
 		}
 	}
 
+	/**
+	 * Implements filtering of orders by created_via value.
+	 */
+	private function set_created_via_args(): void {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		$created_via = sanitize_text_field( wp_unslash( $_GET['_created_via'] ?? '' ) );
+
+		if ( empty( $created_via ) ) {
+			return;
+		}
+
+		$this->order_query_args['created_via'] = explode( ',', $created_via );
```

Comment:
Might be a little bit cautious, but we could map over these and trim just to be safe:

```suggestion
		$this->order_query_args['created_via'] = array_map( 'trim', explode( ',', $created_via ) );
```

Reply:
Good idea, just pushed the change.